docs/_sass/_colors.scss

```
@@ -2,16 +2,16 @@ $color-dark: #0B1B34;
 $color-gray1: #FAFAFA;
 $color-gray2: #ECECEC;
 $color-gray3: #C6C9CA;
-$color-green: #07d973;
+$color-green: #03D47C;
```

```suggestion
$color-green400: #03D47C;
```

if we decide to update color names